RoddyCore/src/de/dkfz/roddy/execution/io/SSHExecutionService.groovy

```diff
@@ -133,15 +133,15 @@ class SSHExecutionService extends RemoteExecutionService {
                 t2 = System.nanoTime();
                 logger.postSometimesInfo(RoddyIOHelperMethods.printTimingInfo("start ssh client session", t1, t2));
             } catch (Exception ex) {
-                logger.severe("Fatal error during initialization of SSHExecutionService. Password-based or password-less key-based authentication supported. No ssh-agent support.")
+                logger.severe("Fatal error during initialization of SSHExecutionService. Message: \"${ex.message}\". Check password-based or password-less key-based authentication to all your head nodes.")
```

Comment:
Could you elaborate on this? What do you mean?

Reply:
More like:
Fatal error during initialization of SSHExecutionService. Message: \"${ex.message}\". 
Please check the following entries in your application ini file:
CLI.executionServiceAuth
CLI.executionServiceHost
CLI.executionServicePasswd ( in case "executionServiceStorePassword" is set to true 
CLI.executionServiceStorePassword
CLI.executionServiceUser=USERNAME